docs/schemas/itemMeta.json

```diff
@@ -478,6 +489,26 @@
         "title": {
           "$ref": "#/definitions/title"
         },
+        "subtitle": {
+          "$ref": "#/definitions/subtitle",
+          "description": "It's only usable when presentation of cascade item is showCards"
+        },
+        "description": {
+          "$ref": "#/definitions/description",
+          "description": "It's only usable when presentation of cascade item is showCards"
+        },
+        "footer": {
+          "$ref": "#/definitions/footer",
+          "description": "It's only usable when presentation of cascade item is showCards"
+        },
+        "scale": {
+          "type": "number",
+          "description": "Start from 0. Determine the scale of cascade choice. Scale is for determining the colour of card component. It's only usable when presentation of cascade item is showCards"
```

Comment:
umm, i'm not totally clear from this description what exactly `scale` is.

Reply:
umm, I think about the same thing too. This attribute used to determine the choice "level", so that UI can render different colours based on their level. For example, the level is 1 then UI renders "blue" for this choice. So maybe I change it to "level?

Reply:
yeah, `level` sound good. "defines the colour of the card components when the presentation of cascade item is showCards. Starts at zero"?